content/docs/tools-ui/ui-dev-messages.md

```diff
@@ -0,0 +1,537 @@
+---
+linktitle: UI Messages
+title: Design Guidelines for UI Messages in Open 3D Engine
+description: Learn about the design models and best practices for O3DE UI messages, including error messages and notifications.
+weight: 500
+toc: true
+---
+
+Are you developing user interface (UI) components that extend **Open 3D Engine (O3DE)**? Learn how to craft meaningful and useful user interface (UI) messages that help your users by reviewing the principles and best practices in this topic. UI messages indicate the status of a component, such as error, failure, warning, information, and success messages. This document covers various types of messaging including *Toast*, *Inline Notification*, *Dialog*, *Console Log*, *Event Log*, *Event Table*, *Component Card*, and *Text Input Validation*.
+
+Consistent styling in O3DE messages is crucial because it reduces the cognitive load on users to know how to resolve issues in future scenarios.
+ 
+## Prerequisites
+
+To aid in your learning, we recommend opening the **Qt Control Gallery**, a Qt widget library that demonstrates UI elements and allows you to program and preview additional elements. The QT Control Gallery is built when you build O3DE and is located in the engine's `build/<platform>/bin/<configuration>` directory.
+
+For more information, refer to [O3DE Qt Control Gallery Tool](/docs/tools-ui/uidev-control-gallery/)
```

Comment:
**What did i change?**
Previous text:
- [Line 15-17]
  > We believe that consistency in O3DE messages is crucial, as it reduces the cognitive load on how issues can be resolved for future scenarios.
  > As a prerequisite, we recommend having access to the [**Qt Control Gallery**](https://o3de.org/docs/tools-ui/uidev-control-gallery/), a pre-built library which allows you to incorporate additional elements, to get started.
- [Line 32]
  > **Related Guide:** [**O3DE Qt Control Gallery Tool**](/docs/tools-ui/uidev-control-gallery/)

Comments:
- I rephrased the first sentence and moved it to the introduction paragraphs.
- I made a separate "Prerequisites" section, which is commonly found throughout the guide to indicate tools or knowledge users should have before continuing to the content in the page. 
- We only need one link to the Qt Control Gallery Tool. I opted to put it at the end in a "For more information..." sentence, to keep the paragraph above free from hyperlinking distraction. 
- Only bold "**Qt Control Gallery**" once -- the first time it appears in a page.
- I mentioned where users can find the Qt Control Gallery. Although this is covered more in detail in the [O3DE Qt Control Gallery Tool](/docs/tools-ui/uidev-control-gallery/) page, it's worth briefly mentioning it here to help users avoid going to another page to find that out.  
- I removed the term "prebuilt library" because it can be confused with `.lib` or `.dll` files, when the application I'm looking for actually has the `.exe` extension, `O3DEQtControlGallery.exe`. I replaced it with "Qt widget library, because that's how it's described in the [O3DE Qt Control Gallery Tool](/docs/tools-ui/uidev-control-gallery/) page.
- Use active voice in the phrase, "how issues can be resolved". Who is resolving the issues, the users or the developers? In this case, the UI messages are helpful from the user-end.